### PR TITLE
Minor initrd improvements

### PR DESCRIPTION
**Pull request by erikarvstedt**

### CODE REVIEW

File: nixos/modules/system/boot/initrd-network.nix [hunk 1]
```diff
@@ -45,7 +45,8 @@ in
         is acquired using DHCP.
 
         You should add the module(s) required for your network card to
-        boot.initrd.availableKernelModules. lspci -v -s &lt;ethernet controller&gt;
+        boot.initrd.availableKernelModules.
+        <literal>lspci -v | grep -iA8 'network\|ethernet'</literal>
```
Comment (teto) [inline nixos/modules/system/boot/initrd-network.nix]: I am not used to the doc but shouldn't it be <command> instead ?

Comment (erikarvstedt) [reply nixos/modules/system/boot/initrd-network.nix]: Be what? :relaxed:

Comment (teto) [reply nixos/modules/system/boot/initrd-network.nix]: github hid the text between < > xD

Comment (erikarvstedt) [reply nixos/modules/system/boot/initrd-network.nix]: Nice catch, fixed.

Comment (erikarvstedt) [reply nixos/modules/system/boot/initrd-network.nix]: After a closer look it seems that NixOS docs and descriptions mostly use `<literal>` for complex shell commands:
```
grep -r '<literal>.*|' $(nix eval --raw '(builtins.fetchTarball { url = https://nixos.org/channels/nixos-unstable/nixexprs.tar.xz; })')
# vs
grep -r '<command>.*|' $(nix eval --raw '(builtins.fetchTarball { url = https://nixos.org/channels/nixos-unstable/nixexprs.tar.xz; })')
```
I've reverted the change.

**Comment by Mic92** [comment]
@GrahamcOfBorg build nixosTests.boot.luksroot

**Comment by erikarvstedt** [comment]
Is there any way to get universal keyboard HID support in initrd?
Using vendor-specific drivers seems rather excessive here.

**Comment by 7c6f434c** [comment]
I think Cherry USB keyboards work with `hid_cherry` but not `hid_generic`, for example…

**Comment by Mic92** [comment]
In theory that should be usbhid, however some devices are not compliant to the usb keyboard device class.

**Comment by erikarvstedt** [comment]
I've removed the extra `initrdAvailableKernelModules` entry and filed https://github.com/NixOS/nixpkgs/issues/52962 instead.

**Pull request merged**